mate-session/gsm-logout-dialog.c

```diff
@@ -464,30 +464,30 @@ gsm_get_dialog (GsmDialogLogoutType type,
                 logout_dialog->default_response = GSM_LOGOUT_RESPONSE_SHUTDOWN;
 
                 if (gsm_logout_supports_system_suspend (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("S_uspend"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("S_uspend"), "system-suspend",
```

Comment:
in my tests, this icon is missing

Reply:
I'll try with another set of icons like "Mate" or "Menta" to find another one.  I search for a icon with name that not begins with "gnome", perhaps a good idea for the future...

Reply:
In debian we have the package `gtk-3-examples` , it installs the  program `gtk3-icon-browser` and you can see a lot of icons

for reference:
https://askubuntu.com/questions/695796/view-list-of-all-available-unique-icons-with-their-names-and-thumbnail

@mate-desktop/core-team the icon names `"system-suspend"` and `"system-hibernate"` work for you? not here

Reply:
Thanks for the link!
Done, switch to another one.